services/storage/tests/unit/test_simcore_s3_dsm.py

```diff
@@ -133,3 +160,131 @@ async def test_upload_and_search(
     for file in files:
         assert file.sha256_checksum == checksum
         assert file.file_name in {"file1", "file2"}
+
+
+@pytest.fixture
+async def paths_for_export(
+    create_empty_directory: Callable[
+        [str, ProjectID, NodeID], Awaitable[SimcoreS3FileID]
+    ],
+    populate_directory: Callable[
+        [ByteSize, str, ProjectID, NodeID, int, int],
+        Awaitable[tuple[NodeID, dict[SimcoreS3FileID, FileIDDict]]],
+    ],
+    delete_directory: Callable[[StorageFileID], Awaitable[None]],
+    project_id: ProjectID,
+    node_id: NodeID,
+) -> AsyncIterable[set[SimcoreS3FileID]]:
+    dir_name = "data_to_export"
+
+    directory_file_upload: SimcoreS3FileID = await create_empty_directory(
```

Comment:
I want to discuss that.
I spent a big amount of time trying to reduce all the variants of these fixtures uploading files and stuff. let's check together.